src/containers/shared/components/Transaction/EscrowCancel/Description.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { useTranslation, Trans } from 'react-i18next'
+import { Link } from 'react-router-dom'
+import { findNode, normalizeAmount } from '../../../transactionUtils'
+import Account from '../../Account'
+import {
+  TransactionDescriptionComponent,
+  TransactionDescriptionProps,
+} from '../types'
+
+const Description: TransactionDescriptionComponent = (
+  props: TransactionDescriptionProps,
+) => {
+  const { t, i18n } = useTranslation()
+  const language = i18n.resolvedLanguage
+  const { data } = props
+  const deleted: any = findNode(data, 'DeletedNode', 'Escrow')
+  if (deleted == null) {
+    return null
+  }
+  return (
+    <>
+      <div>
+        {t('escrow_cancellation_desc')} <Account account={data.tx.Account} />
+      </div>
+      <div>
+        <Trans i18nKey="escrow_cancellation_desc_2">
```

Comment:
Why do these two lines seem to use two different ways of translating?

Reply:
@ckniffen

Reply:
Trans is used when adding elements to the interpolated values of a translation.